modules/ROOT/pages/validate-4-task.adoc

```diff
@@ -1,34 +1,57 @@
-= Validate Query Parameters and Headers Using APIkit
+= Validating Query Parameters and Headers
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-partial:
 
-This process assumes that you have generated flows using APIkit in Studio.
-
-You can configure APIkit Router to validate headers and query parameters to limit both these attributes to those defined in your API specification:
+Configure *APIkit Router* to enforce validation on headers and query parameters, restricting them only to the attributes specified within your API specification.
 
 // tag::process[]
-. In your Studio canvas, select *APIkit Router*.
+. In Studio, select *APIkit Router*.
 +
-image::apikit-router-selection.png["APIkit router highlighted."]
-. Select the *Edit* button for your router configuration
+image::apikit-router-selection.png["APIkit router highlighted"]
+. Click *Edit* for your router configuration.
 +
-image::router-config.png["Edit button highlighted next to the plus button, in the APIkit router tab."]
+image::router-config.png["Edit button highlighted in the APIkit router tab"]
 // end::process[]
-. Select *Query parameters Strict Validations*  to limit query parameters to those defined in your API specification, or *Headers Strict Validations* to limit headers to those defined in your API specification:
+
+. Select *Query Parameters Strict Validations* to restrict the usage of query parameters to only the ones defined in your API specification. 
```

Comment:
Suggest something like:
Specify validation settings:
- Query Parameters Strict Validations
- Headers Strict Validations

Reply:
Thanks for your comment, @hannanelson! 
Your option is shorter and I like it! Nevertheless, it may be confusing for the readers because the explanation lacks the purpose of the configurations, i.e.: to restrict the usage of XXX to the ones defined in the users' API specification.
Or did I misunderstand your comment?
Thanks!

Reply:
I didn't mean to remove the descriptions, just change the format of the list to add a sub-bullet list.

Reply:
I may be misunderstanding your point. But there is no list in this file. There are steps. And each step provides the step itself and its purpose.
Thanks!